### PR TITLE
feat: train cvd with v2

### PR DESCRIPTION
**Pull request by MartinBernstorff**

### CODE REVIEW

File: psycop/common/model_training_v2/trainer/preprocessing/steps/column_validator.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def from_list(
         return cls(prefix=prefix, count=count)  # type: ignore
 
 
-@BaselineRegistry.preprocessing.register("prefix_count_validator")
+@BaselineRegistry.preprocessing.register("column_prefix_count_expectation")
```
Comment (MartinBernstorff) [inline psycop/common/model_training_v2/trainer/preprocessing/steps/column_validator.py:79]: @HLasse I didn't get an error that 'prefix_count_validator' no longer exists in the registries. Perhaps we want the generated placeholders to include addressing? E.g.:

```cfg
[placeholder]
@preprocessing = "prefix_count_validator"
```

File: psycop/projects/cvd/model_training/preprocessing/regex_filter.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+from psycop.common.model_training_v2.config.baseline_registry import BaselineRegistry
+from psycop.common.model_training_v2.trainer.preprocessing.step import (
+    PolarsFrame_T0,
+    PresplitStep,
+)
+
+
+@BaselineRegistry.preprocessing.register("regex_column_blacklist")
+class RegexColumnBlacklist(PresplitStep):
```
Comment (MartinBernstorff) [inline psycop/projects/cvd/model_training/preprocessing/regex_filter.py:11]: @HLasse I've added a few modules here as 'hacks'. Let me know if you think any of them are interesting, then we should polish and add them to the shared code :-)

Comment (MartinBernstorff) [reply psycop/projects/cvd/model_training/preprocessing/regex_filter.py:11]: (see the other files also)

Comment (MartinBernstorff) [reply psycop/projects/cvd/model_training/preprocessing/regex_filter.py:11]: Same for you as well @sarakolding.

File: psycop/projects/cvd/model_training/data_loader/trainval_loader.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+from psycop.common.model_training_v2.config.baseline_registry import BaselineRegistry
+from psycop.common.model_training_v2.trainer.base_dataloader import BaselineDataLoader
+
+
+class MissingPathError(Exception):
+    ...
+
+
+@BaselineRegistry.data.register("parquet_vertical_concatenator")
+class ParquetVerticalConcatenator(BaselineDataLoader):
```
Comment (HLasse) [inline psycop/projects/cvd/model_training/data_loader/trainval_loader.py:16]: I love this way of doing it -- removes a lot of complexity from the data loaders. Strong vote for adding this to common

File: psycop/projects/cvd/model_training/preprocessing/bool_to_int.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+from psycop.common.model_training_v2.config.baseline_registry import BaselineRegistry
+from psycop.common.model_training_v2.trainer.preprocessing.step import (
+    PolarsFrame_T0,
+    PresplitStep,
+)
+
+
+@BaselineRegistry.preprocessing.register("bool_to_int")
+class BoolToInt(PresplitStep):
```
Comment (HLasse) [inline psycop/projects/cvd/model_training/preprocessing/bool_to_int.py:12]: Also seems like a fairly general purpose and generally useful preprocessing step. Simple enough to just move to common imo

File: psycop/projects/cvd/model_training/data_loader/trainval_loader.py (hunk 3 above)
Comment (HLasse) [reply psycop/projects/cvd/model_training/data_loader/trainval_loader.py:16]: (code is nice and simple and naming is good, ok for moving this to main as is from me)

File: psycop/projects/cvd/model_training/preprocessing/datetime_filter.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+from psycop.common.model_training_v2.config.baseline_registry import BaselineRegistry
+from psycop.common.model_training_v2.trainer.preprocessing.step import (
+    PolarsFrame_T0,
+    PresplitStep,
+)
+
+
+@BaselineRegistry.preprocessing.register("temporal_col_filter")
+class TemporalColumnFilter(PresplitStep):
```
Comment (HLasse) [inline psycop/projects/cvd/model_training/preprocessing/datetime_filter.py:11]: Super useful presplitstep -- one vote for common! Had to look `cs.temporal` to understand what a temporal column entails though. Perhaps adding the following as docstring would make it work for common:

"""Removes temporal columns (e.g. of type datetime) from the dataframe"""

File: psycop/projects/cvd/model_training/preprocessing/regex_filter.py (hunk 2 above)
Comment (HLasse) [reply psycop/projects/cvd/model_training/preprocessing/regex_filter.py:11]: I see myself using all of them within the next week's time so I propose moving them all of them to main :)

**Review by HLasse — APPROVED** [review]
This is excellent, Martin! It's great seeing that our re-factoring has worked! Fan of all the functions you have registered - I suggest moving them all to common

**Pull request merged**